test/apis/authentication-api.ts

```diff
@@ -173,32 +165,51 @@ describe('Authentication API', () => {
     assert(currentPatient.lastName == 'Duchâteau')
   }).timeout(60000)
 
-  it('Patient should be able to signing up through email using a different Storage implementation', async () => {
-    // Given
-    const storage: Record<string, string> = {}
+  it('Patient should be able to retrieve its keys when re-login', async () => {
+    // When
+    const { api, user } = await TestUtils.signUpUserUsingEmail(env!.iCureUrl, env!.msgGtwUrl, env!.specId, env!.patAuthProcessId, hcpId!)
 
-    class MemoryStorage implements StorageFacade<string> {
-      async deleteItem(key: string): Promise<void> {
-        delete storage[key]
-      }
+    // Then
+    const currentUser = user
+    assert(currentUser)
+    assert(currentUser.patientId != null)
 
-      async getItem(key: string): Promise<string | undefined> {
-        return storage[key]
-      }
+    const keyPair = await api.initUserCrypto()
```

Comment:
This is not a re-login : Here, you just override the Api of the same session. 
If you want a re-login, you need to re-create a new MedTechApi